acceptance/acceptance_suite_test.go

```diff
@@ -0,0 +1,208 @@
+package acceptance_test
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/codeskyblue/kexec"
+	"github.com/onsi/ginkgo/config"
+	"github.com/suse/carrier/cli/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestAcceptance(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Acceptance Suite")
+}
+
+var nodeSuffix, nodeTmpDir string
+
+var _ = SynchronizedBeforeSuite(func() []byte {
+	if os.Getenv("REGISTRY_USERNAME") == "" || os.Getenv("REGISTRY_PASSWORD") == "" {
+		panic("REGISTRY_USERNAME and REGISTRY_PASSWORD environment variables can't be empty")
+	}
+
+	fmt.Printf("Compiling Carrier on node %d\n", config.GinkgoConfig.ParallelNode)
+
+	buildCarrier()
```

Comment:
It is unclear to me why `carrier` is built here, given that it is also built in the flow (main.yml#L23).